client/blocks/inline-help/index.jsx

```diff
@@ -187,6 +188,11 @@ class InlineHelp extends Component {
 						showNotification={ showChecklistNotification }
 					/>
 				) }
+				{ isWithinBreakpoint( '<660px' ) && isPopoverVisible && (
+					<RootChild>
```

Comment:
Nice TIL that we had this component! ✨  

https://github.com/Automattic/wp-calypso/tree/0fabad68d2189bedae70a4e87dd6fcaf65348349/packages/components/src/root-child